cmd/convert.go

```diff
@@ -123,7 +123,7 @@ func Convert(releaseName string) error {
 		}
 	} else {
 		if !settings.dryRun {
-			fmt.Printf("Release \"%s\" was converted successfully from Helm 2 to Helm 3. Note: the v2 releases still remain and should be removed to avoid conflicts with the migrated v3 releases.\n", releaseName)
+			fmt.Printf("Release \"%s\" was converted successfully from Helm 2 to Helm 3. Note: the v2 release information still remains and should be removed to avoid conflicts with the migrated v3 release.\n", releaseName)
```

Comment:
Can we add a sentence at the end of this message to tell the user how best to remove the v2 release info? Maybe something like this: 'v2 release information should be only be removed using `helm 2to3 cleanup` and when all releases have been migrated over.'

Reply:
done